libs/product/src/models/composite-product-item.ts

```diff
@@ -0,0 +1,17 @@
+import { DaffProduct } from './product';
+
+export interface DaffCompositeProductItem {
+	id: number;
+	required: boolean;
+	title: string;
+	type: string;
+	options: DaffCompositeProductItemOption[];
+}
+
+export interface DaffCompositeProductItemOption extends DaffProduct {
+	can_change_quantity: boolean;
```

Comment:
Since we don't have a use for this yet, lets remove it.

Reply:
removed